Paymetheus/StakeMining.xaml

```diff
@@ -0,0 +1,77 @@
+﻿<Page x:Class="Paymetheus.StakeMining"
+      xmlns:local="clr-namespace:Paymetheus" 
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+      xmlns:vm="clr-namespace:Paymetheus.ViewModels"
+      mc:Ignorable="d"
+      x:Name="Page"
+      WindowTitle="Page"
+      FlowDirection="LeftToRight"
+      WindowWidth="640" WindowHeight="480" d:DesignWidth="640" d:DesignHeight="480"
+      DataContext="{Binding Source={StaticResource ViewModelLocator}, Path=StakeMiningViewModel}" Loaded="Page_Loaded">
+
+    <ScrollViewer Style="{StaticResource MainPaneScrollViewerStyle}">
+        <StackPanel Margin="40 20 40 0">
+            <Grid Margin="0 0 0 0">
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="*"/>
+                    <ColumnDefinition Width="120"/>
+                </Grid.ColumnDefinitions>
+                <TextBlock Margin="0 0 0 10" Text="Stake mining" Grid.Column="0" Style="{StaticResource shellMediumHeaderTextBlockStyle}"/>
+                <Button Grid.Column="1"  HorizontalAlignment="Right" VerticalAlignment="Center" Margin="0 0 0 0"
+                    Content="REFRESH" Command="{Binding FetchStakeInfoCommand}"
+                    Style="{DynamicResource ButtonBlue}"/>
+            </Grid>
+            <Grid Margin="0 40 0 0">
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="225"/>
+                    <ColumnDefinition Width="10"/>
+                    <ColumnDefinition Width="*"/>
+                </Grid.ColumnDefinitions>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="25"/>
+                    <RowDefinition Height="20"/>
+                    <RowDefinition Height="20"/>
+                    <RowDefinition Height="20"/>
+                    <RowDefinition Height="20"/>
+                    <RowDefinition Height="25"/>
+                    <RowDefinition Height="20"/>
+                    <RowDefinition Height="20"/>
+                    <RowDefinition Height="20"/>
+                    <RowDefinition Height="20"/>
+                    <RowDefinition Height="20"/>
+                    <RowDefinition Height="20"/>
+                    <RowDefinition Height="20"/>
+                    <RowDefinition Height="20"/>
+                </Grid.RowDefinitions>
+                <!-- Network statistics -->
+                <TextBlock Margin="0 0 0 0" Text="Network statistics" FontWeight="Bold"  Style="{StaticResource shellDescriptionTextBlockStyle}" HorizontalAlignment="Left" VerticalAlignment="Top" Grid.Row="0" Grid.Column="0"/>
```

Comment:
I agree with removing the margin setters and making most of the heights of the rows auto, but don't agree with declaring a new style element since we're already using shellDescriptionTextBlockStyle. I prefer to have the width of the grid remain static so that if the user resizes the window, the information and layout of the table will not be disrupted by spanning widely across the GUI.